src/DIRAC/RequestManagementSystem/private/RequestValidator.py

```diff
@@ -1,8 +1,3 @@
-########################################################################
-# File: RequestValidator.py
-# Author: Krzysztof.Ciba@NOSPAMgmail.com
-# Date: 2012/09/18 07:55:16
-########################################################################
 """ :mod: RequestValidator
```

Comment:
The `RequestValidator` needs to work with both versions, because it is instantiated on the client and the server side, so the tests should cover both cases

Reply:
Check what I have done for covering this case.